### ISSUE TITLE
q37_FossilFeedstock_Base still relevant?

### ISSUE THREAD

**Issue by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [remindmodel/remind#1606]
https://github.com/remindmodel/remind/blob/25fb451b542efce1c817554e2f796d702eb3b3f1/modules/37_industry/subsectors/equations.gms#L324-L334

> that was set up by @fschreyer to prevent the model from using biomass or synfuels in baseline runs. However, I think that now that might be unnecessary because we improved emissions accounting (including incineration) and we are not supposed to use `baseline` anymore...

https://github.com/remindmodel/remind/pull/1599#issuecomment-1980448924

Does this still have some relevance?  If so, we need to add further restrictions.

**Comment by fschreyer** [comment]
How do Npi feedstock sources look now? Are they all going non-fossil?

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
Fossil/biomass/synfuels mix for feedstocks is identical to that of the chemicals subsector overall.

**Comment by fschreyer** [comment]
Ah ok, good! And that is not going crazy compared to results before the feedstocks implementation in terms of a shift to non-fossil sources, right? I guess, the equation can then simply be removed without any need for further action.

**Comment by fschreyer** [comment]
I meant the distribution of fossil, biogenic, synthetic sources non-energy FE. So this one, for example from ``/p/projects/remind/runs/REMIND_2024_02_16/remind/compScen-SSP2EU-2024-02-19_18.54.49-H12.pdf`` :

Looks all pretty fossil, which at least for the Npi scenario makes sense. There is a weird hydrogen spike in 2100 for Npi but that's a [known issue](https://github.com/remindmodel/development_issues/issues/214) for all of REMIND and I am working on an update which will fix this.  

In summary, the equation is not needed anymore. 

![image](https://github.com/remindmodel/remind/assets/42831977/62effa3a-f36b-4c22-a3f8-284c03bfd40d)

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
Well, there is no plot for feedstocks composition before the feedstocks implementation.
Your plot on the other hand is before the fix, so the data is rather meaningless.

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
2024-02-23
![image](https://github.com/remindmodel/remind/assets/53254462/a470ae2e-f0a5-4300-a3ca-893a800f2ef3)

**Comment by fschreyer** [comment]
> Your plot on the other hand is before the fix, so the data is rather meaningless.

Well, the probability that all of industry goes fossil in Npi when before the constraint both industry and non-energy use went fossil is quite high I'd say. But we will see.
